data/recipes/gcp_cloud_resource_tree_offline.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gcp_cloud_resource_tree_offline",
+  "description": "Generates a parent/children tree for given GCP resource using the supplied exported GCP logs",
+  "short_description": "Generates a parent/children tree for given GCP resource using the supplied exported GCP logs",
+  "modules": [{
+      "wants": [],
+      "name": "FilesystemCollector",
+      "args": {
+          "paths": "@paths"
+      }
+  }, {
+      "wants": ["FilesystemCollector"],
+      "name": "GCPCloudResourceTree",
+      "args": {
+          "project_id": "@project_id",
+          "resource_name": "@resource_name",
+          "resource_type": "@resource_type",
+          "mode": "offline"
+      }
+  }],
+  "args": [
+      ["project_id", "ID of the project where the resource is located", null],
+      ["resource_name","Resource name", null],
+      ["resource_type","Resource type (currently supported types: gce_instance, gce_disk, gce_image, gce_machine_image, gce_instance_template, gce_snapshot)", null],
+      ["paths", "Comma-separated paths to GCP log files", null]
```

Comment:
add the expected file format, ex: json, to the description on the argument

Reply:
Done